protocol-dashboard/src/components/UniqueUsersChart/UniqueUsersChart.tsx

```diff
@@ -47,8 +50,8 @@ const UniqueUsersChart: React.FC<UniqueUsersChartProps> = () => {
   return (
     <LineChart
       titleTooltipComponent={UniqueUsersInfoTooltip}
-      topNumber={topNumber}
-      title='Unique Users'
+      title={`Unique ${BUCKET_TO_TOPLINE_LABEL[bucket]} Users`}
+      topNumber={data ? data[data.length - 1] : undefined}
```

Comment:
Might be worth making a constant for the data cause it's a bit hard to understand this line